src/pkg/bundle/remove.go

```diff
@@ -106,8 +152,21 @@ func removePackages(packagesToRemove []types.Package, b *Bundle) error {
 			if err := pkgClient.Remove(context.TODO()); err != nil {
 				return err
```

Comment:
if we run into an error while removing, is it ok for the state to remain state.Removing?

Reply:
We could add another state called `FailedRemove`

Reply:
👍 added